receiver/gitproviderreceiver/go.mod

```diff
@@ -4,8 +4,8 @@ go 1.20
 
 require (
 	github.com/Khan/genqlient v0.6.0
-	github.com/google/go-cmp v0.5.9
-	github.com/google/go-github/v53 v53.2.0
+	github.com/google/go-cmp v0.6.0
```

Comment:
Where did this version change come from? I thought this was just go-github bump?

Reply:
I ran 3 commands to prep the changes
```
make tidy-all
make fmt-all
make lint-all
```
That must have come from `make tidy-all`